### PR TITLE
Added SeqBRNN support

### PR DESCRIPTION
**Pull request by SeanNaren**
Copy of request from [here](https://github.com/nicholas-leonard/rnn/pull/3). It also seems like the issue with _grad_x has been solved.

Let me know of any issues/feedback!

### CODE REVIEW

File: SeqBRNN.lua [hunk 1]
```diff
@@ -0,0 +1,77 @@
+------------------------------------------------------------------------
+--[[ BRNN ]] --
```
Comment (nicholas-leonard) [inline SeqBRNN.lua]: BRNN -> SeqBRNN

batch x time x inputdim -> time x batch x inputdim

**Comment by nicholas-leonard** [comment]
@SeanNaren Looks great! Could you add some README doc for the two modules? doesn't need to be extensive.

**Comment by SeanNaren** [comment]
Done! Hopefully everything is fixed and the docs are explanatory enough, let me know. 

Also the readme will need a merge, should be straight forward!

**Comment by nicholas-leonard** [comment]
@SeanNaren Thanks! Looks great.

**Pull request merged**